cloudevents/sdk/__init__.py

```diff
@@ -0,0 +1,16 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+# Convenience aliases
+from cloudevents.sdk.event.base import MarshallerType, UnmarshallerType  # noqa: F401
```

Comment:
I think these should just be moved here instead (although I would prefer them to be in `cloudevents/types/__init__.py`)

Reply:
How about `cloudevents/sdk/types.py`? It feels strange to have the type definitions at a higher level than the rest of the SDK.

Reply:
I think the `sdk` submodule is actually the strange part... this whole library is the "sdk" 🙂 But that's probably a good compromise.

Reply:
I agree that it's strange to have the library under "sdk", but I don't want to rock the boat in this PR.

Reply:
I appreciate that! I agree we can make that a separate issue.